var/spack/repos/builtin/packages/iozone/package.py

```diff
@@ -22,6 +23,8 @@ class Iozone(MakefilePackage):
     build_directory = 'src/current'
 
     def edit(self, spec, prefix):
+        chmod = which('chmod')
+        chmod('-R', '755', self.stage.source_path)
```

Comment:
Can we use Python [os.chmod](https://docs.python.org/3/library/os.html#os.chmod) instead?

Reply:
I see how this is preferred but noticed the heavy usage of ` which('chmod')` within other packages. Fixed now.